### PR TITLE
replacing "..." (3 dots) with "…" (ellipsis)

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
see https://musescore.org/en/node/280844

It is quite a large change...

### CODE REVIEW

File: mscore/startcenter.ui [hunk 1]
```diff
@@ -106,8 +106,6 @@
    <container>1</container>
   </customwidget>
  </customwidgets>
- <resources>
-  <include location="musescore.qrc"/>
- </resources>
+ <resources/>
```
Comment (mattmcclinch) [inline mscore/startcenter.ui]: Was this intentional? It seems unrelated to the purpose of the PR.

Comment (Jojo-Schmitz) [reply mscore/startcenter.ui]: A side effect that I left in deliberatly.

**Comment by Jojo-Schmitz** [comment]
Please check the discussion in https://musescore.org/en/node/280844, also several translators use it in there translations and some files issues for it on Transifex.
I'm not a huge fan, but don't have a strong dislike either.

**Comment by riaanvn** [comment]
We have discussed the change at
https://musescore.org/en/node/280844
To summarise
a) many translators expect ellipsis when they see the 3 dots and translate it as such
b) 3 dots is considered a hack by some
c) the style guide of other larger projects specify explicitly using the ellipsis character (incl [Mozilla]( 
https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practice))
d) If the translator wanting to be correct with the convention "out there" uses the ellipsis and the original has three dots, it complicates comparison between the two when the tool doing the comparison does not find the same punctuation. 

I will document ellipsis usage in our style guide, once merged.

My hope is that when the PR is merged, the old strings are removed and the new strings are shown as untranslated, that the translation memory will remember the old translation and offer it as a suggestion. I do not have 100% proof that this is the case when changes were made to strings. 

Jojo: If the translators are not offered their old translations as suggestions, is there any way we can offer translators their language files to get the old translations (rather than expect them to each export via Transifex? Since all old translations contain 3 dots or ellipsis, they should be very easy to filter on.

The file Library/Application\ Support/MuseScore/MuseScore3/locale/ contains the binary .qm which one can only read using Qt Linguist. Looking for a .ts that contains the translated strings. 

I have also asked the fellow translators if any of them have any objections against or experience with ellipsis vs 3 dots at
https://musescore.org/en/node/281367
and received no feedback.

**Comment by Jojo-Schmitz** [comment]
It might be possible to pills all ts files from Transifex, batch convert ... to ellipsis and push back to Transifex?

**Comment by riaanvn** [comment]
I am sure it is possible. It depends on
a) how much work it is per language (or if someone can do it globally if it can be done in bulk). The big advantage from what I have seen is that it is only in mscore, not in any of the other resources.
b) what permissions are needed.
c) who does the work. I am more than willing to help do the work as long as I have the necessary permissions and don't do it for individual strings. I am willing to accept that I may have to do it for individual language (if it cannot be avoided)

I am more than willing for you to use Afrikaans as a guinea pig. 

Also, I just had an idea - is there any way we can test if the Suggestion / translation memory will remember and suggest the pre-ellipsis translation of a string when that string has been deleted? I was thinking if you commit a single string change in a separate PR, we can look at what suggestions if any come through for it. If it does not offer the old translation, it means we have a big more work to do on our side (e.g. make sure translators have access to the 3 dot versions of the translations, or even decide to the export/replace/import on our end).

String that came to mind (still has 3 dots)
Show &apos;More Elements...&apos;
in Transifex as
Show 'More Elements...'
Complex enough that it won't match closely with anything other than this string

**Comment by riaanvn** [comment]
Hi Jojo

I just realised - we can use the strings in [various text fixes #4492](https://github.com/musescore/MuseScore/pull/4492)  as a test to see if changing an existing string will suggest the translation of the old/removed string. [replacing "..." (3 dots) with "…" (ellipsis) #4527](https://github.com/musescore/MuseScore/pull/4527).

**Comment by anatoly-os** [comment]
I think I can manage this. I will manually (using a script, ofc) update all actual .ts files and push them to Transifex. So that our translators should do nothing regarding this PR.

**Comment by anatoly-os** [comment]
Successfully replaced ... with … locally using this PR and manual tx pull/push.
